Libraries/Components/View/ReactNativeStyleAttributes.js

```diff
@@ -47,4 +48,16 @@ ReactNativeStyleAttributes.shadowColor = colorAttributes;
 ReactNativeStyleAttributes.textDecorationColor = colorAttributes;
 ReactNativeStyleAttributes.tintColor = colorAttributes;
 
+if (Platform.OS === 'android') {
+  // ReactAndroid doesn't currently have the capability to have variable typing
+  // (eg. string or array) of a native attribute.  To work around that, we can
+  // transform variable types into one standard type, as done here with
+  // textDecorationLine which can be an array of enum values or a space-separated
+  // string containing the same values.
+  var processTextDecorationLine = require('processTextDecorationLine');
```

Comment:
We're excluding the module atm and throwing an error on runtime. I'd like the packager to have as little platform resolution code as possible, so I'm happy with the current behavior. Otherwise we'll need to account for this on only on the packager but on all the tooling that is around it and react-native (flow, jest, etc) :)